boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -112,7 +112,10 @@ else
 fi
 
 if [[ "$generate_script" = "common" ]] ; then
-    ./boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py -o ${operator_name} -d ${OUTPUT_DIR} -p ${OPERATOR_PREV_VERSION} -i ${REPO_DIGEST} -V ${operator_version}
+    if [[ -z "$CONTAINER_ENGINE" ]]; then
+        CONTAINER_ENGINE=$(command -v podman || command -v docker)
+    fi
+    $CONTAINER_ENGINE run -it --rm -v `pwd`:`pwd` -u `id -u`:0 -w `pwd` registry.access.redhat.com/ubi8/python-36:1-134 /bin/bash -c "python -m pip install oyaml; python ./boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py -o ${operator_name} -d ${OUTPUT_DIR} -p ${OPERATOR_PREV_VERSION} -i ${REPO_DIGEST} -V ${operator_version}"
```

Comment:
You need to make that many adjustments for permissions? Is that to run locally or to work in jenkins?

Reply:
Locally. I have *no idea* what it's going to take to run in jenkins. But without the permission stuff I couldn't a) pip install (gid has to be 0 for that), or b) lay down files e.g. the saas bundle clone (uid has to be what's on the files to begin with).